books/principles-management/bake

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+require_relative '../../bake_helper'
+
+gemfile do
+  gem 'openstax_kitchen', \
+      ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : KITCHEN_VERSIONS[:principles_management]
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :principles_management) do |doc|
+  include Kitchen::Directions
+
+  # Set overrides
+  doc.selectors.override(
+    page_summary: 'section.section-summary'
+  )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v2(
+    book: book,
+    strategy_options: {
+      strategy: :default, bake_chapter_outline: false, introduction_order: :v1
+    }
+  )
+  BakeChapterTitle.v1(book: book)
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[exploring-manager-careers concept-check
+                sustainability-responsible managerial-leadership
+                ethics-in-practice managing-change
+                catching-spirit expanding-around-globe customer-satisfaction]
+  )
+
+  BakeReferences.v1(book: book, metadata_source: metadata)
+
+  book.chapters.each do |chapter|
+    BakeLearningObjectives.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
+
+    BakeChapterReferences.v1(chapter: chapter, metadata_source: metadata)
+
+    eoc_sections = %w[section-summary chapter-review manage-skills
+                      manage-exercises critical-thinking]
+
+    chapter.figures(only: :figure_to_number?).each do |figure|
+      BakeFigure.v1(figure: figure,
+                    number: "#{chapter.count_in(:book)}.#{figure.count_in(:chapter)}")
+    end
+
+    chapter.tables('$:not(.unnumbered)').each do |table|
+      BakeNumberedTable.v2(table: table,
+                           number: "#{chapter.count_in(:book)}.#{table.count_in(:chapter)}")
+    end
+
+    eoc_sections.each do |section_key|
+      MoveCustomSectionToEocContainer.v1(
+        chapter: chapter,
+        metadata_source: metadata,
+        container_key: section_key,
+        uuid_key: ".#{section_key}",
+        section_selector: "section.#{section_key}"
+      ) do |section|
+        RemoveSectionTitle.v1(section: section)
+      end
```

Comment:
generally I think it's better to bake numbered figures & tables after all the sections are moved, so that the numbering goes in order of where the figures & tables appear. (they didn't do this in easybake, so it does create numbering differences in the diff from easybake, which is annoying)